### PR TITLE
Step7-2

### PR DESCRIPTION
**Pull request by Ayana326**
## What
I deleted the branch of step7 after merging by mistake....
### Demo
Build_Mercari
### New API to delete items
I wanted to delete duplicated items to take video.
So, I created a new API to delete items.
If you run `curl -X GET 'http://127.0.0.1:9000/items/1'`, an item with id=1 will be deleted.
But this command only removes the column from the items table, not the categories table...
## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -111,6 +111,11 @@ def add_item_to_json(new_item):
     with open(file_path, "w") as file:
         json.dump({"items": items_list}, file)
 
+# 商品の削除
+@app.get("/delete/{item_id}")
```
Comment (tsbkw) [inline python/main.py]: Usually, for deleting single resource, DELETE is used. Also the path is the same as GET (=`/items/{item_id}`).
Sometimes, POST method is used for deletion by expressing delete in path.
GET is almost always used for getting data, and if you modify or delete resources by GET.


DELETE METHOD: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE 
Also, Safe section of GET METHOD is good to read. https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET

If you have time, the course referred in Step3 will help.
> [Udemy Business - REST WebAPI サービス 設計](https://mercari.udemy.com/course/rest-webapi-development/)

File: python/main.py [hunk 2]
```diff
@@ -271,4 +276,13 @@ def select_join_items():
...
 
-    return {"items": items_list}
\ No newline at end of file
+    return {"items": items_list}
+
+def delete_item(item_id):
+    conn = sqlite3.connect(db/"items.db")
+    cur = conn.cursor()
+
+    cur.execute('DELETE FROM items WHERE id = ?', (item_id,))
```
Comment (tsbkw) [inline python/main.py:297]: This is totally depends on your intention and design but if you want to delete category when item is deleted, you can.
> But this command only removes the column from the items table, not the categories table...

One option is get category_id of the target item to delete and if there are no items which use the same category, you can delete it from category table.

In practice, category is usually defined by service provider (for example Mercari), and user can't define own category. Therefore, we don't have to care about category deletion when item is deleted.

**Review by tsbkw — COMMENTED** [review]
Good challenge! I commented so please take a look.

**Comment by Ayana326** [comment]
Thank you for your advice!
I could implement delete API maybe!!

File: python/main.py (hunk 2 above)
Comment (Ayana326) [reply python/main.py:297]: Thank you for your advice!
I could implement delete API maybe!!

**Pull request merged**